test/run-drun/try-finally.mo

```diff
@@ -313,6 +315,21 @@ actor A {
         try await t6t() catch _ {};
         try await t6d() catch _ {};
         try await t8t() catch _ {};
+
+        // other side problem tests
+        try await async {
```

Comment:
Ok, I think this is behaving correctly. Maybe refactor into a separate subtest/function (like the others)

Reply:
I'll do this in a separate PR.